x-pack/plugins/security_solution/cypress/tasks/timeline.ts

```diff
@@ -207,7 +207,10 @@ export const openTimelineTemplateFromSettings = (id: string) => {
 };
 
 export const openTimelineById = (timelineId: string) => {
-  return cy.get(TIMELINE_TITLE_BY_ID(timelineId)).pipe(($el) => $el.trigger('click'));
+  return cy
+    .get(TIMELINE_TITLE_BY_ID(timelineId))
+    .pipe(($el) => $el.trigger('click'))
+    .should('be.visible');
```

Comment:
If I'm not mistaken, the above `pipe` will be clicking on the  `TIMELINE_TITLE_BY_ID(timelineId)` element till it becomes visible. But if we are clicking on it, it is already visible. That means the `pipe` function will never be repeated. Should we check another element for visibility inside `should`? Like next page's title?